metabase/metabase
# Remove " (ms)" from "Query Runs" label in Enterprise admin section

Fixes #16714 

## How to Test

* In Enterprise, go to Admin.
* Click on `Audit`.
* Click on `Questions`.
* Click on `All Questions`.

You should no longer see the ` (ms)` substring in the label.

![image](https://user-images.githubusercontent.com/380816/122951133-6da1d200-d353-11eb-9802-4e48f89d339d.png)
